build/linux/installer/scripts/tomlparser-osm-config.rb

```diff
@@ -151,7 +151,7 @@ def replaceOsmTelegrafConfigPlaceHolders
     telemetryFile = File.open("integration_osm_config_env_var", "w")
 
     if !telemetryFile.nil?
-      telemetryFile.write("export TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT=#{@osmMetricNamespaces.length}\n")
+      telemetryFile.write("export OSM_CONFIGURATION_NAMESPACES_COUNT=#{@osmMetricNamespaces.length}\n")
```

Comment:
@jatakiajanvi12 , can you please double check this name change made across all the places and not missing any other place? we can also revert the name change if this name changes are not required.

Reply:
Found one more location. Made changes for the same